### DISCUSSION TITLE
We are unable to complete your signup. Please read this guide for further details.

### DISCUSSION THREAD

**Discussion by shivharerachit** [vercel/community#3786] [Account Issue] [answered]
### Question

We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.
account not logging in using github

**Comment by amyegan** [comment]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please check for an email with more details about the error and instructions on next steps. For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by invessa** [comment]
Hi @amyegan ,
I am getting this same error when I tried to signup. I have not received any follow up email from Vercel, telling me what the issue is with my signup. Can you help? Can I send you my account details ?

**Reply by amyegan** [reply, marked as answer]
Since we can't safely discussion account info in public, please use the [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) for login issues found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Reply by alexander0205** [reply]
@invessa Did you get any solution?

**Comment by yousefeltrach** [comment]
i have the same problem any solution
